code/modules/mob/living/simple_animal/abnormality/aleph/seasons.dm

```diff
@@ -884,4 +1230,323 @@
 /obj/effect/thunderbolt/seasons/Convert(mob/living/carbon/human/H) //haha, it doesn't actually convert.
 	return
 
+/obj/structure/fire_wall
+	gender = PLURAL
+	name = "Mini Vulcano"
+	desc = "It's spewing fire!"
+	icon = 'icons/effects/effects.dmi'
+	icon_state = "summer_attack"
+	anchored = TRUE
+	density = TRUE
+	max_integrity = 10000
+	armor = list(
+		MELEE = 0,
+		BULLET = 0,
+		FIRE = 100,
+		RED_DAMAGE = 100,
+		WHITE_DAMAGE = 190,
+		BLACK_DAMAGE = 100,
+		PALE_DAMAGE = 100,
+	)
```

Comment:
Why not save a bunch of lines of code by making this indestructible? And why does it need to be dense?